scripts/build-controller-image.sh

```diff
@@ -79,6 +88,11 @@ if ! is_public_ecr_logged_in; then
   aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
 fi
 
+pushd "$ROOT_DIR" 1>/dev/null
+  # Get the golang version from the code-generator
+  GOLANG_VERSION=${GOLANG_VERSION:-"$(go list -f {{.GoVersion}} -m)"}
```

Comment:
So if i understand well, now controllers will be built using a Go compiler version parsed from code-generator `go.mod`? Meaning that if we bump the version in the go.mod, it will bump the compiler version?

Reply:
That's correct. We need to be using the minimum Golang version as the code-generator, which should also be the same minimum version as the runtime.